### ISSUE TITLE
Accommodate tumor cell receptor ligand interactions in cytokine network display

### ISSUE THREAD

**Issue by vthorsson** [CRI-iAtlas/shiny-iatlas#160]
Allow tumor cells to be included in extracellular network module if results are available

**Comment by heimannch** [comment]
Do we have an annotation of all receptors and ligands that interact with tumor cells (perhaps from the cell-image module)?

**Comment by vthorsson** [comment]
Indeed, from the cell-image diagram we have some examples of these to work from. 
The attached can serve as an example file to start from (text "Tumor_cell" can be changed to whatever is most appropriate for input )

[CellImage_InteractionPairs.tsv.gz](https://github.com/CRI-iAtlas/shiny-iatlas/files/3957569/CellImage_InteractionPairs.tsv.gz)
